### DISCUSSION TITLE
Solidity Comment extension

### DISCUSSION THREAD

**Discussion by Tezei** [Cyfrin/foundry-full-course-cu#2786] [General]
Hiya guys,
Can you please point me to the VS Code NatSpec comment extension that Patrick uses?

10x in advance

**Comment by EngrPips** [comment]
I am not sure I remember correctly what VS Code NatSpec comment extension he used but maybe this will work too 

<img width="1148" alt="Screenshot 2024-10-31 at 18 26 55" src="https://github.com/user-attachments/assets/b125eb7f-72ec-4f81-ae80-bc3e674aa09b">

**Reply by Tezei** [reply]
10x

**Reply by EngrPips** [reply]
Anytime.
